torch/onnx/_internal/fx/passes/fx_to_onnxscript.py

```diff
@@ -125,10 +133,33 @@ def _retrieve_or_adapt_input_to_graph_set(fx_node_arg, fx_name_to_onnxscipt_valu
         # 2. fx_node_arg (variable in torch.fx.Graph) is be mapped to
         #    torch.jit.Value, fx_name_to_onnxscipt_value[fx_node_arg.name],
         #    in TorchScript graph.
-        onnx_tensor = fx_name_to_onnxscipt_value[onnx_tensor.name]
-    elif isinstance(onnx_tensor, torch.dtype):
+        return fx_name_to_onnxscipt_value[onnx_tensor.name]
+    if isinstance(onnx_tensor, (tuple, list)) and any(
+        isinstance(ele, torch.fx.Node) for ele in onnx_tensor
```

Comment:
Are we sure we want to concatenate all `lists of fx.Node`?

Reply:
yeah we need to be careful about this

Reply:
What if there is a `[Tensor<bool>, Tensor<float>]` (e.g., dropout_backward's inputs)? There can be other forward ops with hybrid element types.

Reply:
Make sense

Reply:
Done